game.py

```diff
@@ -25,9 +25,14 @@
 # Get starting state of the game from the backend module.
 state = backend.get_start_state(data)
 
+# Constants for frontend module.
+TILE_WIDTH = data["tilewidth"]
+TILE_HEIGHT = data["tileheight"]
+WINDOW_WIDTH = data["width"]*TILE_WIDTH
+WINDOW_HEIGHT = data["height"]*TILE_HEIGHT
```

Comment:
Když se to mění s nově načtenou mapou, tak už to není konstanta, ale informace o mapě (nebo něco co na informacích o mapě závisí).

Kam načítáte informace o mapě?

Reply:
Ok tak už v tom vidím ten rozdíl, takže to, co jsem tam přidala nejsou konstanty a ty proměnné měly by být napsané malými písmeny. 

Z JSONu načítáme informace o mapě do proměnné 'data'.